docs/faststore/docs/faststore-api/schema/mutations.mdx

```diff
@@ -44,6 +44,8 @@ Web session identifier.
 </tbody>
 </table>
 
+> ⚠️ There is a Known Issue with the `vvalidateCart` mutation. If an external request modifies the store cart with additional information like marketing data, it disrupts the store cart synchronization process, interfering with the connection between the cart displayed on the storefront and the actual cart data stored on the server. This may lead to issues for store users, such as items not being added, incorrect totals, or checkout problems. After any external cart update, refresh the cart data using the workaround instructions in the [Cart does not update in FastStore if an external request is made to update orderForm](https://help.vtex.com/en/known-issues/cart-does-not-update-in-faststore-if-external-request-is-made-to-update-orderform--7ef1GxxapbH2XKKf7HBuAM) guide.
```

Comment:
```suggestion
> ⚠️ There is a Known Issue with the `validateCart` mutation. If an external request modifies the store cart with additional information like marketing data, it disrupts the store cart synchronization process, interfering with the connection between the cart displayed on the storefront and the actual cart data stored on the server. This may lead to issues for store users, such as items not being added, incorrect totals, or checkout problems. After any external cart update, refresh the cart data using the workaround instructions in the [Cart does not update in FastStore if an external request is made to update orderForm](https://help.vtex.com/en/known-issues/cart-does-not-update-in-faststore-if-external-request-is-made-to-update-orderform--7ef1GxxapbH2XKKf7HBuAM) guide.
```